package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-toggle",
-  "version": "2.2.1",
+  "version": "2.3.0",
```

Comment:
This should be 2.2.2 since it's a patch, not a feature. Fix that and I'll merge :)